views/problem_display.ejs

```diff
@@ -37,11 +37,11 @@
                 <h2>Output</h2>
                 <pre><%= data.outputFormat %></pre>
                 <h2>Constraints</h2>
-                <pre><%= data.constraints %></pre>
-                <h2>Sample Input</h2>
-                <pre><%= data.sampleInput %></pre>
-                <h2>Sample Output</h2>
-                <pre><%= data.sampleOutput %></pre>
+                <pre><%= data.constraints %></pre> 
+                <h2>Sample Input<span><button class="border border-dark ml-4 p-2 btn btn-light btn-sm" onclick="copyToClipboard(one)"><b>Copy text to clipboard</b></button></span></h2>
+                <pre id = "one"><%= data.sampleInput %></pre>
+                <h2>Sample Output<span><button class="border border-dark ml-4 p-2 btn btn-light btn-sm" onclick="copyToClipboard(one)"><b>Copy text to clipboard</b></h2>
```

Comment:
don't you think it should be `copyToClipboard(two)` instead of `copyToClipboard(one)`

Also it will be better if you write the id of these pre tags as something better like _sample-input-copy_ and _sample-output-copy_